portfolio/src/main/webapp/quiz.js

```diff
@@ -2,40 +2,91 @@
  * Question data type.
  */
 class Question {
-  constructor(question, answers, correct) {
+  constructor(question, answers, correctAnswerIndex) {
     this.question_ = question;
     this.answers_ = answers;
     this.correctAnswerIndex_ = correctAnswerIndex;
   }
 
   getQuestion() {
-    return question_;
+    return this.question_;
   }
 
   getAnswers() {
-    return answers_;
+    return this.answers_;
   }
 
   getCorrectAnswerIndex() {
-    return correctAnswerIndex_;
+    return this.correctAnswerIndex_;
   }
 }
 
+// create array of quiz questions 
+const questions = [];
```

Comment:
Optional: What about questionsArray and in that way we don't need the comment above?

Reply:
Yea that's a good idea, I'll do that.